rotkehlchen/tests/exchanges/test_binance.py

```diff
@@ -873,7 +874,12 @@ def test_binance_query_trade_history_custom_markets(function_scope_binance):
     binance = function_scope_binance
 
     markets = ['ETHBTC', 'BNBBTC', 'BTCUSDC']
-    binance.edit_exchange(name=None, api_key=None, api_secret=None, PAIRS=markets)
+    binance.edit_exchange(
+        name=None,
+        api_key=None,
+        api_secret=None,
+        **{BINANCE_MARKETS_KEY: markets},
```

Comment:
A variable cannot be used as a kwarg I suppose. That was the reason for `**{BINANCE_MARKETS_KEY: markets}`.
I could make it cleaner by doing...
```py
kwargs = {BINANCE_MARKETS_KEY: markets}
binance.edit_exchange(name=None, api_key=None, api_secret=None, **kwargs)
```

Reply:
You can try it yourself, is also what was used before

```
>>> def test_f(b, **kwargs):
...     print(b, kwargs)
... 
>>> test_f(1, BINANCE_MARKETS_KEY=[3])
1 {'BINANCE_MARKETS_KEY': [3]}
>>> 
```

Reply:
As you have it now it should work since you also changed the function argument but can you just use
```suggestion
        binance_selected_trade_pairs=markets,
```

It improves readability a lot and also is easier to detect errors by the linters

Reply:
Alright then. Thanks